introduction/how-to-get-help/revised-community-guideline/README.md

```diff
@@ -41,7 +42,16 @@ Engaging in the programming community requires adopting a set of principles that
 3. **Giving and Receiving**: Embrace mentorship and offer assistance to fellow programmers, fostering a culture of support.
 4. **Continuous Learning**: Develop a growth mindset by learning from peers and seeking constructive feedback.
 
+## Graduate attribute for Web development professionals
+
+While going through your web developpment courses, students will exposed to a range of attributes that are of highly valued in the programming community across the globe. And are some important aspects for the web development professionals. It is important that you take note of these values and focus to build up on these skills as you approach your professional path in the area of web development. Use the discord community and intiate conversations about these topics with your fellow learners.
```

Comment:
@vsharma2288 Possible to include a hyperlink where they can read more about attributes if they wish to?

Reply:
Have added the reference link.